.teamcity/_self/projects/WebApp.kt

```diff
@@ -459,6 +459,10 @@ fun playwrightPrBuildType( targetDevice: String, buildUuid: String ): BuildType
 			cleanCheckout = true
 		}
 
+		params {
+			param("env.SAVE_AUTH_COOKIES", "true")
+		}
```

Comment:
Nit: you can also use a `checkbox` instead of asking the user to type "true" or "false" (that being said, we have a ton of parameters that could be checkboxes but aren't, so this is not a new problem. Feel free to ignore, we can change this one if/when we change the rest of the boolean parameters.

```suggestion
		params {
				checkbox(
					name = "env.SAVE_AUTH_COOKIES",
					value = "true",
					label = "Save authentication cookies",
					description = "Login once and reuse auth cookies for all specs",
					checked = "true",
					unchecked = "false"
				)
			}
		}
```

Reply:
Neat, I did not know about this!

Reply:
Fixed in 8d1fe0d0de45743c639026ebc118f29237f5f07c.